bucket/displayfusion.json

```
@@ -0,0 +1,38 @@
+{
+    "version": "9.5",
+    "description": "Multi-monitor desktop managing tool",
+    "homepage": "https://www.displayfusion.com/",
+    "license": {
+        "identifier": "Freeware",
+        "url": "https://www.displayfusion.com/License/"
+    },
+    "url": "https://binaryfortressdownloads.com/Download/BFSFiles/101/DisplayFusionSetup-9.5.exe#/setup.exe",
```

That is why you have `$fname`
```suggestion
    "url": "https://binaryfortressdownloads.com/Download/BFSFiles/101/DisplayFusionSetup-9.5.exe",
```

---

Sorry. Didn't notice that.